src/main/java/org/minions/devfund/alejandro/movie/database/Movie.java

```diff
@@ -0,0 +1,78 @@
+package org.minions.devfund.alejandro.movie.database;
+
+import java.util.ArrayList;
+
+/**
+ * This class contains the information about the movies.
+ */
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * This method is the constructor for movie class.
+     * @param name The name of the movie.
+     */
+    public Movie(final String name) {
+        this.name = name;
+        this.actors = new ArrayList<Actor>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)